src/main/java/step2/Cal.java

```diff
@@ -0,0 +1,119 @@
+package step2;
+
+import java.util.Arrays;
+
+public class Cal {
+
+    private static String[] array = {"+","-","*","/"};
+
+    public String[] calsachicInput(String[] splitresult){
+        System.out.println("*****************************");
+        System.out.println("사칙연산저장");
+        String[] result = new String[splitresult.length];
+        for(int i = 0 ;i<splitresult.length;i++) {
+            if (splitresult[i].equals("+")) {
+                result[i] = "+";
+                System.out.println(result[i]);
+            }
+            else if(splitresult[i].equals("-")){
+                result[i] = "-";
+                System.out.println(result[i]);
+            }
+            else if(splitresult[i].equals("*")){
+                result[i] = "*";
+                System.out.println(result[i]);
+            }
+            else if(splitresult[i].equals("/")){
+                result[i] = "/";
+                System.out.println(result[i]);
+            }
+        }
+        result = getoutNull(result);
+        return result;
+    }
+
+  //  String sampleword ="2 + 3 * 4 / 2";
+
+    public String[] splitWord(String sampleword){
+        sampleword =  sampleword.replaceAll(" ", "");
+        String[] arr = new String[sampleword.length()];
+
+        for (int i = 0; i < sampleword.length(); i++) {
+            arr[i] = sampleword.substring(i,i+1);
+
+        }
+
+
+        return arr;
+    }
+
+    public String[] extraction(String[] splitresult) {
+        System.out.println("*****************************");
+        String[] result = new String[splitresult.length];
+        for(int i = 0 ;i<splitresult.length;i++) {
+            if (!splitresult[i].equals("+") && !splitresult[i].equals("-") &&
+                    !splitresult[i].equals("*")  && !splitresult[i].equals("/") ){
+                result[i] = splitresult[i];
+            }
+
+        }
+        result =  getoutNull(result);
+
+        for (int i = 0; i < result.length; i++) {
+            System.err.println(result[i]);
+        }
+        return result;
+    }
+
+
+    public String docal(String[] sachicresult, String[] numberresult) {
```

Comment:
```suggestion
    public String doCal(String[] sachicresult, String[] numberresult) {
```
자바 스타일 가이드를 참고하시면 좋을 것 같아요 👍 
https://google.github.io/styleguide/javaguide.html#s5.3-camel-case